css/style.css

```diff
@@ -0,0 +1,68 @@
+   body{
```

Comment:
- Required linter check for style sheet

Reply:
- [ ] It is always nice to make your comments suggestive or having polite phrases, instead of commanding sentences. In addition, you should always write comments highlighting the line or block of code related to your comments. I don't think linter has anything to do with a line of code in the CSS file.